include/psen_scan_v2/scanner_v2.h

```diff
@@ -47,7 +47,10 @@ static constexpr std::chrono::milliseconds REPLY_TIMEOUT{ 1000 };
 class ScannerV2 : public IScanner
 {
 public:
-  ScannerV2(const ScannerConfiguration& scanner_config, const LaserScanCallback& laser_scan_cb);
+  ScannerV2(const ScannerConfiguration& scanner_config,
+            const LaserScanCallback& laser_scan_cb,
+            const unsigned short data_port_scanner = DATA_PORT_OF_SCANNER_DEVICE,
```

Comment:
Could/Should these be part of the ScannerConfiguration?

Reply:
I'm actually not sure. If so, the `ScannerConfiguration` should have default values for it because it is probably only rarely changing.